inference-engine/tests/unit/inference_engine/ie_blob_proxy.cpp

```diff
@@ -86,20 +79,19 @@ TEST_F(BlobProxyTests, canAccessProxyBlobUsingBaseMethod)
     TBlobProxy<float> proxy(Precision::FP32, C, spBlob, 2, {2});
 
     auto proxyBuffer = proxy.buffer();
-    float *ptr = (float*)(void*)proxyBuffer;
+    float *ptr = proxyBuffer.as<float*>();
```

Comment:
**It's not a blocker for merge, but just an advice if you will refactor any code which relates to buffer usage in the future**: maybe it makes sense to move to the usage of `r`\\`w`\\`map` conception for `LockedMemory` creation instead of buffers?

Reply:
it's for the future)

Reply:
Ok, as you wish, it was just a nice idea to take in account for the future.